cameras/paths.csv

```diff
@@ -32,3 +33,4 @@ Arecont,camera,AV Models,(00:1A:07),rtsp://{{username}}:{{password}}@{{ip_addres
 Microseven,camera,M7XXX,(10:83:D2),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/1{{stream}},H.264,554,1;2;3,,High;Medium;Low,,3,1,admin,password,,http://www.microseven.com/manuals/M7xxx-NetworkCamera-UserGuide_2016_FINAL.pdf
 Grandstream,nvr,GXV3504,(00:0B:82),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{channel}},H.264,554,,0;1;2;3;4;5;6;7,,Channel {{channel}},,,admin,admin,,http://www.voipsupply.com/downloads/dl/file/id/8911/grandstream_gxv3504_manual.pdf
 Grandstream,camera,GXV3674_[F]HD_VF,(00:0B:82),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}},H.264,554,4,,Stream {{stream}},,4,4,admin,admin,,http://www.grandstream.com/sites/default/files/Resources/GXV3674hd_fhd_v2_usermanual_english.pdf
+Vivotek,camera,CC8130,(00:02:D1),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/{{stream}}.sdp,H.264,554,1;2;3,,live;live2;liveany;,,live2,live,,,,http://download.vivotek.com/downloadfile/downloads/usersmanuals/cc8130manual_en.pdf
```

Comment:
The `streams` have to be `live;live2;liveany` as the actual stream values (not just the descriptions).